__tests__/app.test.js

```diff
@@ -5,13 +5,55 @@ const data = require('../db/data/test-data')
 const db = require('../db/connection')
 const endpoints = require('../endpoints.json')
 
-beforeEach(() => {
-    return db.query('DELETE FROM comments;')
-        .then(() => db.query('DELETE FROM articles;'))
-        .then(() => seed(data))
-})
+beforeEach(() => seed(data))
 afterAll(() => db.end());
 
+describe('/api/articles/:article_id/comments', () => {
+    test('GET: 200 - responds with an array of comments for the given article_id of which each comment should have the following properties:', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then((result) => {
+                expect(Array.isArray(result.body.comments)).toBe(true)
```

Comment:
I would add a test for the length of the array before using the forEach. If the array is empty then the tests inside your forEach would be skipped but length test would still run